src/test/java/dk/kb/present/ViewTest.java

```diff
@@ -62,6 +62,20 @@ void jsonldPvica() throws Exception {
         assertTrue(jsonld.contains("\"name\":\"Før Bjørnen Er Skudt\""));
     }
 
+    @Test
+    void solrFromPvica() throws Exception {
+        YAML conf = YAML.resolveLayeredConfigs("test_setup.yaml");
+        YAML radiotvConf = conf.getYAMLList(".config.collections").get(1);
+        View solrView = new View(radiotvConf.getSubMap("radiotv").getYAMLList("views").get(2), radiotvConf.getSubMap("radiotv").getString("origin"));
+        String pvica = Resolver.resolveUTF8String("internal_test_files/tvMetadata/df3dc9cf-43f6-4a8a-8909-de8b0fb7bd00.xml");
```

Comment:
There's an awful lot of "we know what the setup file contains"-magic here and in the other tests. We should consider a more resilient approach. But not in this pull request.,

Reply:
These are to be known as "YAML incantations"